modules/nw-rfhe-creating-hardware-event.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * networking/using-rfhe.adoc
+
+:_content-type: PROCEDURE
+[id="nw-rfhe-creating-hardware-event_{context}"]
+= Creating the Redfish HardwareEvent and Secret CRs
+
+To start using baremetal hardware events, create the a `HardwareEvent` custom resource (CR) for the host where the Redfish hardware is present. Hardware events and faults are reported in the `hw-event-proxy` logs.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Install the Baremetal Hardware Event Proxy Operator.
+* Create a `BMCEventSubscription` CR for the BMC Redfish hardware.
+
+[NOTE]
+====
+Create a single instance of the `HardwareEvent` resource. Multiple `HardwareEvent` resources are not permitted.
```

Comment:
Can this note be condensed?  I feel like the second sentence being first eliminates the need for the first sentence. 
For example, 

Multiple `HardwareEvent` resources are not permitted.

Reply:
applied.